istio.io/configmap-nginx.yaml

```diff
@@ -149,6 +149,47 @@ data:
           return 301 https://istio.io$request_uri;
       }
 
+      # gcsweb.istio.io
+      # Upgrade HTTP to HTTPS
+      server {
+        server_name gcsweb.istio.io;
+        listen 80;
+
+        location = /_healthz {
+          add_header Content-Type text/plain;
+          return 200 "ok\n";
+        }
+
+        return 301 https://$server_name$request_uri;
+      }
+
+      # Handle HTTPS.
+      upstream gcsweb {
+              server    gcsweb.default.svc.cluster.local:80;
+        keepalive 10;
+      }
+
+      server {
+        server_name gcsweb.istio.io;
```

Comment:
We'll need to add gcsweb.istio.io SAN to istio.io cert before this goes live.

Reply:
How do we do that ?

Reply:
Done